app/questionnaire/questionnaire_schema.py

```diff
@@ -116,6 +121,47 @@ def placeholder_transform_section_dependencies_by_block(
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    @cached_property
+    def min_and_max_map(self) -> Any:
+        return make_immutable(self._min_and_max_map)
+
+    def _create_min_max_map(
+        self,
+        min_max: Literal["minimum", "maximum"],
+        answer_id: str,
+        answers: list[ImmutableDict],
+        default_min_max: str,
+    ) -> None:
+        longest_value_length: int = 0
```

Comment:
non blocker, happened to just notice, you don't need the int type hint here as mypy will infer it